ui/packages/catalog-ui-search/src/main/webapp/component/result-form/result-form.collection.js

```diff
@@ -42,26 +68,51 @@
      })
    }],
    addResultForms: function () {
+    this.set('doneLoading',false)
      if (!this.isDestroyed) {
-       let filteredList = Results.getResultTemplatesProperties().filter(function (resultField) {
-         return resultField.id !== 'allFields'
-       })
-       filteredList.forEach(element => {
-         let utcSeconds = element.created / 1000
-         let d = new Date(0)
-         d.setUTCSeconds(utcSeconds)
-         this.addResultForm(new ResultForm({
-           createdOn: Common.getHumanReadableDate(d),
-           id: element.id,
-           name: element.label,
-           type: 'result',
-           descriptors: element.descriptors,
-           accessIndividuals: element.accessIndividuals,
-           accessGroups: element.accessGroups,
-           description: element.description
-         }))
-       })
-     };
+      if (promiseIsResolved === true) {
+        promiseIsResolved = false;
+        bootstrapPromise = new resultTemplatePromise();
+      }
+      bootstrapPromise.then(() => {
+            this.filteredList = _.map(resultTemplates, function(resultForm) {
+              return {
+                  label: resultForm.title,
+                  value:resultForm.title,
+                  id: resultForm.id,
+                  descriptors: resultForm.descriptors,
+                  description: resultForm.description,
+                  created: resultForm.created,
+                  creator: resultForm.creator,
+                  createdBy: resultForm.owner,
```

Comment:
❓ I think we want this to be `resultForm.creator` if we want to align with [search forms](https://github.com/codice/ddf/blob/a9c99e9c74b1cc7a31d82927e22c48080ffc751c/ui/packages/catalog-ui-search/src/main/webapp/component/search-form/search-form.collection.js#L117)